app/api/resources/task.py

```diff
@@ -0,0 +1,232 @@
+from flask import request
+from flask_restx import Resource, Namespace, marshal
+from flask_jwt_extended import jwt_required, get_jwt_identity
+from http import HTTPStatus
+from app import messages
+from app.api.dao.task import TaskDAO
+from app.api.dao.task_comment import TaskCommentDAO
+from app.api.resources.common import auth_header_parser
+from app.api.dao.task import MentorshipRelationDAO
+from app.api.dao.user import UserDAO
+from app.api.models.task import *
+from app.api.validations.task_comment import (
+    validate_task_comment_request_data,
+    COMMENT_MAX_LENGTH,
+)
+from app.utils.validation_utils import get_length_validation_error_message
+from app.database.models.task import MentorshipRelationModel
+from app.api.email_utils import send_email_task_accepted
+
+
+task_ns = Namespace(
+    "Tasks",
+    description="Operations related to " " tasks in a mentorship relation " "between users",
```

Comment:
```suggestion
    description="Operations related to tasks in a mentorship relation between users",
```
I know this part was copied and pasted, but could you please correct this?